login_server/src/cpp/SingletonManager/EmailManager.cpp

```diff
@@ -141,6 +141,33 @@ int EmailManager::ThreadFunction()
 				mailClientSession.sendMessage(mailMessage);
 					email_sended = true;
 				}
+				catch (Poco::Net::SSLConnectionUnexpectedlyClosedException& ex) {
+					// it is a mad idea to send an email if the email sending failed
+					// errors.sendErrorsAsEmail();
+					// better wait instead and try again, it seems that the strato mail server sometimes discard or connection
```

Comment:
I would assume that this connection trouble is a problem on our side - strato is a professional hoster and certainly will be able to handle many connections without running into bootlnecks. It could be a rate-limit thing that blocks our service, since it sends too many EMail? This would be an interesting topic to research.

Reply:
Yes and than it has maybe a cooldown and because of that I have choosen 5 minutes a relativ long time